server/data/quotes.json

```diff
@@ -56,8 +56,8 @@
     "sentence": "One of the benefits of being bicultural is simply the awareness that how you live is not the only way."
   },
   {
-    "author": "Johann Wolfgang von Goethe",
-    "sentence": "He who knows no foreign languages knows nothing of his own."
+    "author": "George Carlin",
+    "sentence": "“Meow” means “woof” in cat.”"
```

Comment:
It doesn't match the contribution requirement. We need quotes regarding "learning a foreign language" or similar. You need to change it.